app/assets/stylesheets/mixins/_forum_post.scss

```diff
@@ -0,0 +1,38 @@
+@mixin forum_post {
+  .post {
+    // In case the previous post was too short.
+    clear: both;
+
+    // Spacing between posts.
+    margin-bottom: 1em;
+
+    .user {
+      margin-right: 1em;
+      text-align: right;
+      width: 7em;
+      float: left;
+    }
+
+    .contents {
+      padding: 20px;
+      line-height: 1;
+      border: 1px solid #ddd;
+      border-radius: 4px;
+      box-shadow: 0 1px 1px rgba(0, 0, 0, 0.075);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.075)` should only be used in variable declarations; they should be referred to via variable everywhere else.